tests/resources/mlflow-test-plugin/mlflow_test_plugin/dummy_evaluator.py

```diff
@@ -23,7 +23,9 @@ def _load_content_from_file(self, local_artifact_path):
         return pdf.to_numpy()
 
 
+# pylint: disable=attribute-defined-outside-init
 class DummyEvaluator(ModelEvaluator):
+    is_baseline_model = False
```

Comment:
Is this line needed?

Reply:
Yes, there are tests in evaluation that tests passing in baseline model and candidate model and test if evaluation behavior is preserved.

Reply:
Remove